schemas/Manual.locations.schema.json

```diff
@@ -46,56 +42,59 @@
                     },
                     "uniqueItems": true
                 },
-                "place_item_category": {
-                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
+                "region": {
+                    "description": "(Optional) The name of the Region this location is part of.",
+                    "type": "string"
+                },
+                "place_item": {
+                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item_category"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "place_item": {
-                    "description": "(Optional) Specify the list of item that can be placed at this location (will only randomly choose 1).",
+                "dont_place_item": {
+                    "description": "(Optional) Specify the list of items that cannot be placed at this location.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "dont_place_item_category": {
-                    "description": "(Optional) Specify the categories of items that will never be placed at this location.",
+                "place_item_category": {
+                    "description": "(Optional) Specify the category of item that will be placed at this location. (Ignore Logic)",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item_category"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
-                "dont_place_item": {
-                    "description": "(Optional) Specify the list of items that cannot be placed at this location.",
+                "dont_place_item_category": {
+                    "description": "(Optional) Specify the categories of items that will never be placed at this location.",
                     "type": "array",
                     "items": {
-                        "$ref": "#/definitions/Place_item"
+                        "type": "string"
                     },
                     "minItems": 0,
                     "uniqueItems": true
                 },
                 "victory": {
                     "description": "(Optional) Is this location the goal of this Manual Apworld",
```

Comment:
Should we change this to "one of the goal(s)" since now multiple goals are supported?

Reply:
Yep, good catch! Changed that to "one of the possible goals".